packages/js/client-config-builder/src/ClientConfigBuilder.ts

```diff
@@ -52,8 +53,10 @@ export class ClientConfigBuilder extends BaseClientConfigBuilder {
                   ...this.buildWrappers(),
                   ...this.buildPackages(),
                 ]),
-                ...this._config.resolvers,
-                new ExtendableUriResolver(),
+                new RetryResolver(new ExtendableUriResolver(), {
```

Comment:
Not sure about this being a part of the default resolver setup.
Feels very custom to me.